bazel.el

```diff
@@ -88,8 +88,17 @@
           "https://github.com/bazelbuild/buildtools/tree/master/buildifier")
   :risky t)
 
-;;;; Commands to run Buildifier.
+(defvar bazel--magic-comment-rx
```

Comment:
This should be a defconst as users aren't expected to change it.

Reply:
Indeed.  Fixed.